spec/features/parks/05_parks_destroy_spec.rb

```diff
@@ -6,9 +6,26 @@
 
 feature "authorized user can delete park information" do
   scenario "admin signs in, navigated to edits page and deletes park" do
-    park = FactoryGirl.create(:park)
+    park = create(:park)
+    user = create(:user, role: 'admin')
+
+    visit root_path
+    sign_in_as(user)
+
     visit park_path(park)
     click_link("Delete")
     expect(page).to have_content("You have deleted park successfully!")
   end
+
+  scenario "member signs in, should be unable to delete park" do
+    park = create(:park)
+    user = create(:user)
+
+    visit root_path
+    sign_in_as(user)
+
+    visit park_path(park)
+    expect(page).to_not have_content("Delete")
+  end
+
 end
```

Comment:
Use `let` statements for resources shared between tests (i.e. variables that are being defined the exact same, like `park`

```
feature "..." do
  let(:park) { create :park }

  scenario "admin signs in, ..."
    ...
    visit park_path(park)
    ...
  end

  scenario "member signs in, ..."
    ...
    visit park_path(park)
```